### DISCUSSION TITLE
Github Mobile does not receive the notification for a security 2FA sometimes

### DISCUSSION THREAD

**Discussion by cpirtea-bun** [community/community#119684] [Mobile]
### Select Topic Area

Bug

### Body

If the phone was locked and I request 2FA to the Github Mobile app then I receive the notification in the Android notifications area, but when I click it I get an empty notification page.

If I unlock the phone, open the Github Mobile app, then request 2FA, then it always works.

**Comment by stevepopovich** [comment]
Hi @cpirtea-bun thanks for writing in, sorry to hear about this. Can you share the version of the app you are on, and the device and OS version? Also, is this consistent behavior for you?

**Reply by cpirtea-bun** [reply]
Github mobile v1.157.0
Android 14
Samsung Galaxy S23+
All updates applied 

Yes, behaviour is consistent.

**Reply by stevepopovich** [reply]
Ok, thank you. I filed an issue internally to look at this. Sorry for the inconvenience!

**Reply by stevepopovich** [reply]
@cpirtea-bun are you still seeing this issue?

**Comment by Lashen1227** [comment]
Start by checking your device's notification settings and ensuring the app has the necessary permissions. Update the app to the latest version, restart both the app and your device, and consider reinstalling GitHub Mobile if the issue persists. If none of these steps work, contacting GitHub support for further assistance may be necessary.
